src/app/components/TopBanner/CarouselItem.tsx

```diff
@@ -185,7 +187,16 @@ export default function CarouselItem(props: CarouselItemProps) {
         tabIndex={active ? 0 : -1}
         section={section}
       >
-        {(!invisible || intersecting) && <BannerImage alt={banner.title} {...imageSrcSet} />}
+        {(!invisible || intersecting) && (
+          <BannerImage
+            alt={banner.title}
+            onLoad={() => {
+              setIsLoaded(true);
+            }}
+            isLoaded={isLoaded}
```

Comment:
```suggestion
            visible={isLoaded}
```

prop 키를 직관적인 네이밍으로 변경하는게 어떨까요?